### PR TITLE
Je/ucr es percent col

### PR DESCRIPTION
**Pull request by emord**
based off of https://github.com/dimagi/commcare-hq/pull/14522. https://github.com/dimagi/commcare-hq/commit/c6efa17b436f1d2ccd3d350c4a2103272e221cef is the only commit of interest

@benrudolph

### CODE REVIEW

**Comment by emord** [comment]
@czue any interest in expanding the test outside of enikshay's UCRs? After this, the only things that I expect don't work are date based columns and advanced sorting and the `LocationColumn`

**Comment by czue** [comment]
> @czue any interest in expanding the test outside of enikshay's UCRs? After this, the only things that I expect don't work are date based columns and advanced sorting and the LocationColumn

yeah definitely. let me know if you want some help making that happen. imagine you could probably ping the ucr skype or email lists for volunteers

File: corehq/apps/userreports/es/data_source.py [hunk 1]
```diff
@@ -219,29 +220,33 @@ def _get_total_aggregated_results(self):
 
     @method_decorator(catch_and_raise_exceptions)
     def get_total_row(self):
-        total_results = self._get_total_aggregated_results()
+        def _clean_total_row(aggregations, aggregation_name):
+            agg = getattr(aggregations, aggregation_name)
+            if hasattr(agg, 'value'):
+                return agg.value
+            elif hasattr(agg, 'doc_count'):
+                return agg.doc_count
```
Comment (benrudolph) [inline corehq/apps/userreports/es/data_source.py]: just curious what is the separation for the two attributes? does ES return different attr names based on the agg?

**Review by benrudolph — APPROVED** [review]
nice

File: corehq/apps/userreports/es/data_source.py (hunk 1 above)
Comment (emord) [inline corehq/apps/userreports/es/data_source.py]: Aggregations like sum or min use 'value' where as filtered aggregations (used in expanded columns) use doc_count. Ideally, this should use a method on the column's config to interpret the results, but that'll take a small refactor. Will add it to the list of things to clean up

**Pull request merged**